.kitchen.yml

```diff
@@ -22,12 +22,14 @@ suites:
   - name: install
     run_list:
       - recipe[cerner_splunk_ingredient_test::splunk]
-      - recipe[cerner_splunk_ingredient_test::universal_forwarder]
   - name: uninstall
     run_list:
-      - recipe[cerner_splunk_ingredient_test::splunk]
```

Comment:
Ah, I see.  Yeah, you just modified the test recipes.  That's fine, I just hadn't made it that far through the review yet.